### PR TITLE
[README] Add conversion options, update documentation a bit

### PR DESCRIPTION
**Pull request by FredKSchott**

### CODE REVIEW

**Comment by FredKSchott** [comment]
@justinfagnani bah, the documentation instructs the user to install from npm. We need to start publishing modulizer upgrades as a part of our batch of polymer elements publishing.

File: README.md [hunk 1]
```diff
@@ -7,21 +7,24 @@ polymer-modulizer is a tool to convert packages of HTML imports to JavaScript mo
...
+To track our status on Polymer elements, check out or [elements status tracker](./docs/polymer-3-element-status.md).  
+For a list of all breaking changes, see our [Breaking Changes Changelog](./docs/breaking_changes.md).
+
 Good luck, the future is exciting! 😎 Please join us for discussion in the [`#modulizer` Slack channel](https://polymer.slack.com/messages/G6R11FXEC/).
 
+
 ## Overview
 
-polymer-modulizer is designed to convert HTML Imports, and especially those containing Polymer and Polymer elements, to JavaScript modules as automatically as possible.
+polymer-modulizer is designed to convert HTML documents, and especially those containing Polymer and Polymer elements, to JavaScript modules as automatically as possible.
```
Comment (justinfagnani) [inline README.md]: Well, if we change this to "documents" to more accurately reflect that we convert top-level HTML files, we probably want to change "to JavaScript modules" too.

File: README.md [hunk 2]
```diff
@@ -97,6 +100,18 @@ modulizer owner/repo owner2/repo2
 
 This will create a `modulizer_workspace` directory and checkout the repos and their Bower dependencies and convert them all in place. You can then run `polymer serve` in the workspace directory and try out the results in Chrome 61 or Safari 10.1 (or Edge and Firefox with the appropriate flags turned on).
 
+
+## Conversion Options
+
+#### `--import-style ["name"|"path"]`
+
+Setting the import style allows you to set whether dependency imports are specifiered by npm package name, or relative file path. Importing specifiers that use package names are easier for third-party packages to work with, but unlike paths they currently can not run natively on the web. Defaults to "path".
```
Comment (justinfagnani) [inline README.md]: I'd replace "dependency" with "JavaScript"

Comment (justinfagnani) [inline README.md]: "specifiered" -> "specified"

File: README.md [hunk 3]
```diff
@@ -97,6 +100,18 @@ modulizer owner/repo owner2/repo2
...
+
+#### `--add-import-path`
+
+If included, the static `importPath` property will be added to converted Polymer elements. See [the `importPath` documentation](https://www.polymer-project.org/2.0/docs/devguide/dom-template) for more information.
+
```
Comment (justinfagnani) [inline README.md]: Add TODO (HTML comment) for other options

File: README.md (hunk 1 above)
Comment (FredKSchott) [reply README.md]: not sure I get this comment, change "to JavaScript modules" to what?

File: README.md (hunk 3 above)
Comment (FredKSchott) [reply README.md]: These were the only two options that seemed to matter enough to be worth documenting, vs. just being helpful CLI flags for automating our own workflow. Maybe add `--flat` & `--private` as well?

**Pull request merged**